src/utils/Constants.ts

```diff
@@ -1,8 +1,8 @@
 const constants = {
-  user: 'setUsername',
-  pass:  'setPassword',
-  success: 'loginSuccess',
-  fail: 'loginFailed'
+  USER: 'setUsername',
+  PASS:  'setPassword',
+  SUCCESS: 'loginSuccess',
+  FAIL: 'loginFailed'
```

Comment:
while this looks more sleek ( having short constants), you might need to be more descriptive when youre  looking at the reducer -- ex. success could mean a successful anything

feel free to use the naming conventions like: SET_USERNAME, SET_PASSWORD, LOGIN_FAILED etc